File: PRAK205/PRAK205-2210817310013-RyanMuhammadIrfan.c
#include <stdio.h>
#include <math.h>

int main()
{
    int A, B, C, K, L;

    scanf("%d", &A);
    scanf("%d", &B);

    C = sqrt(B * B - A * A);
    K = A + B + C;
    L = C * A / 2;

    printf("\n");
    printf("Alas = %d cm\n", C);
    printf("Tinggi = %d cm\n", A);
    printf("Keliling = %d cm\n", K);
    printf("Luas = %d cm^2\n", L);
}

File: PRAK205/PRAK205-2210817310013-Ryan Muhammad Irfan.py
A = int(input(""))
B = int(input(""))

C = (B**2 - A**2)**0.5

K = A + B + C
L = C * A / 2

print("")
print("Alas = ", format(C, '.0f'), "cm")
print("Tinggi = ", A, "cm")
print("Keliling = ", format(K, '.0f'), "cm")
print("Luas = ", format(L, '.0f'), "cm^2")
